mongo/results.go

```diff
@@ -91,7 +93,11 @@ type UpdateResult struct {
 }
 
 // UnmarshalBSON implements the bson.Unmarshaler interface.
+//
+// Deprecated: Unmarshaling an UpdateResult directly from BSON is not suported and may produce
```

Comment:
```suggestion
// Deprecated: Unmarshalling an UpdateResult directly from BSON is not suported and may produce
```

Reply:
I couldn't apply this and Benji's change through Github, but I've updated the spelling of that line.

For a little extra background, both spellings are technically correct, but the Go driver uses "unmarshalling" more than "unmarshaling", so that seems like a good change. Most of the Go documentation actually uses "unmarshaling", but they can basically be used interchangeably.

Reply:
Oh I didn't know that!  Thanks